### DISCUSSION TITLE
Set Home Position

### DISCUSSION THREAD

**Discussion by MicroSur** [Jyers/Marlin#1428] [Q&A] [answered]
3D touch device.

Menu / Prepare / Homing / Set Home Position.
What does this function do?
 
Home all.
I would like the carriage to arrive at the left-bottom corner after determining the home position.
But "bl touch" feature goto bed center and stay here (

**Comment by Cal-LF** [comment, marked as answer]
If you do a search in the discussions area, there are several posts regarding home position. #1018 #966 #835.

**Comment by MicroSur** [comment]
Set Home Position function doing nothing for normal printing.
And I need to change firmware config to get what I want.
Thanks.
